core/src/main/scala/chisel3/package.scala

```diff
@@ -250,23 +250,22 @@ package object chisel3 {
       * val f1 = 30.2 // Scala float type.
       * val pable = cf"w1 = $w1%x f1 = $f1%2.2f. This is 100%% clear"
       *
-      * // pable is as follows
+      * // the val `pable` is equivalent to the following
       * // Printables(List(PString(w1 = ), Hexadecimal(UInt<5>(20)), PString( f1 = ), PString(30.20), PString(. This is 100), Percent, PString( clear)))
       * }}}
-      *
-      * @throws UnknownFormatConversionException
+      * throws UnknownFormatConversionException
       *         if literal percent not escaped with % or if the format specifier is not supported
       *         for the specific type
       *
-      * @throws StringContext.InvalidEscapeException
+      * throws StringContext.InvalidEscapeException
       *         if a `parts` string contains a backslash (`\`) character
       *         that does not start a valid escape sequence.
       *
-      * @throws IllegalArgumentException
+      * throws IllegalArgumentException
       *         if the number of `parts` in the enclosing `StringContext` does not exceed
       *         the number of arguments `arg` by exactly 1.
       */
-    @nowarn("msg=checkLengths in class StringContext is deprecated")
+    @nowarn("msg=checkLengths in class StringContext is deprecated:s")
```

Comment:
Does adding the `:s` actually do anything here? I thought `nowarn` is always `:s`?